src/app/components/route-block/index.js

```diff
@@ -77,23 +77,26 @@ export function RouteBlock() {
 											{ blockType.keywords.join( ', ' ) }
 										</div>
 									) }
-									<div className="bb-route-block__supports bb-route-block__row">
-										<div>
-											<h2>Supports</h2>
+									{ !! Object.keys( blockType.supports )
```

Comment:
Should we do something like `blockType.supports ?? {}` for cases where the supports is undefined?

Reply:
```suggestion
									{ !! Object.keys( blockType.supports ?? {} )
```